stacktrace_external_test.go

```diff
@@ -112,125 +122,71 @@ func TestNewStacktrace(t *testing.T) {
 func TestExtractStacktrace(t *testing.T) {
 	tests := map[string]struct {
 		f    func() error
+		skip bool
 		want *sentry.Stacktrace
 	}{
 		// https://github.com/pkg/errors
-		"pkg/errors": {RedPkgErrorsRanger, &sentry.Stacktrace{
+		"pkg/errors": {RedPkgErrorsRanger, true, &sentry.Stacktrace{
 			Frames: []sentry.Frame{
 				{
 					Function: "RedPkgErrorsRanger",
 					Module:   "github.com/getsentry/sentry-go_test",
-					Lineno:   29,
+					Lineno:   30,
 					InApp:    true,
 				},
 				{
 					Function: "BluePkgErrorsRanger",
 					Module:   "github.com/getsentry/sentry-go_test",
-					Lineno:   33,
+					Lineno:   34,
 					InApp:    true,
 				},
 			},
 		}},
 		// https://github.com/pingcap/errors
-		"pingcap/errors": {RedPingcapErrorsRanger, &sentry.Stacktrace{
+		"pingcap/errors": {RedPingcapErrorsRanger, true, &sentry.Stacktrace{
 			Frames: []sentry.Frame{
 				{
 					Function: "RedPingcapErrorsRanger",
 					Module:   "github.com/getsentry/sentry-go_test",
-					Lineno:   37,
+					Lineno:   38,
 					InApp:    true,
 				},
 				{
 					Function: "BluePingcapErrorsRanger",
 					Module:   "github.com/getsentry/sentry-go_test",
-					Lineno:   41,
+					Lineno:   42,
 					InApp:    true,
 				},
 			},
 		}},
 		// https://github.com/go-errors/errors
-		"go-errors/errors": {RedGoErrorsRanger, &sentry.Stacktrace{
+		"go-errors/errors": {RedGoErrorsRanger, false, &sentry.Stacktrace{
 			Frames: []sentry.Frame{
 				{
 					Function: "RedGoErrorsRanger",
 					Module:   "github.com/getsentry/sentry-go_test",
-					Lineno:   45,
+					Lineno:   46,
 					InApp:    true,
 				},
 				{
 					Function: "BlueGoErrorsRanger",
 					Module:   "github.com/getsentry/sentry-go_test",
-					Lineno:   49,
+					Lineno:   50,
 					InApp:    true,
 				},
 			},
 		}},
-		// FIXME: The tests below are commented out to avoid introducing a
```

Comment:
I saw this comment, but it seems like instead of leaving the test out, it would be best to keep it and remove them all at once in the future if the desire is to move to their own module.